cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/DefaultTicketRegistry.java

```diff
@@ -47,7 +47,7 @@
  * @since 3.0.0
  */
 @Component("defaultTicketRegistry")
-public final class DefaultTicketRegistry extends AbstractTicketRegistry implements Job {
+public final class DefaultTicketRegistry extends AbstractCrypticTicketRegistry implements Job {
```

Comment:
Most important reason is, consistency in the inheritance hierarchy. I want to be able to control a ticket registry line better by ensuring everything is part of the same "category" in cases where it is allowed. 

Encryption is off by default. You get exactly the same behavior you have today.